worldengine/gui/view.py

```diff
@@ -113,7 +111,3 @@ def draw_land_on_screen(world, canvas):
                 canvas.setPixel(x, y, land_color)
             else:
                 canvas.setPixel(x, y, ocean_color)
-
-
-def draw_ancientmap_on_screen(world, canvas):
```

Comment:
I think I was planning to use it, but I had not use it in the GUI. Anyway we are going to refactor the GUI